metrics/README.md

```diff
@@ -0,0 +1,90 @@
+# Common metrics export interfaces for Knative
```

Comment:
We didn't discuss this, but I think that everything we discussed for metrics should apply to tracing as well.  

Are folks ok with that?

cc @mdemirhan @anniefu @evankanderson @grantr

Reply:
Yep! Though in order of implementation, I would say metrics should definitely go first.